### DISCUSSION TITLE
Custom Playable Character

### DISCUSSION THREAD

**Discussion by sam0003fnf** [FunkinCrew/Funkin#3849] [Q&A] [answered]
how do you add a playable character in v0.5.3? I've tried looking for tutorials but they're all outdated

**Comment by AbnormalPoof** [comment, marked as answer]
[Creating a Playable Character (Funkin' Modding Docs)](https://funkincrew.github.io/funkin-modding-docs/05-custom-playable-characters/05-02-creating-a-playable-character.html)
